### PR TITLE
Feature/#9 add auth

### PR DESCRIPTION
**Pull request by snaveevans**
I forgot to add #9 to my commits ☹️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,32 @@
 # Honey-Dos
 
+## Prerequisites
```
Comment (snaveevans) [inline README.md]: @jantaylor I'd like your input on these prerequisites, is linux the same?

File: .vscode/launch.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
            "cwd": "${workspaceRoot}/HoneyDo.Web",
            "stopAtEntry": false,
            "launchBrowser": {
-               "enabled": true,
+               "enabled": false,
```
Comment (snaveevans) [inline .vscode/launch.json]: This is a personal preference, but I find it very annoying that it opens the browser every time I launch or re-launch the app.

**Comment by snaveevans** [comment]
Need to run `dotnet restore`, `npm install --prefix HoneyDo.Web/ClientApp` and `dotnet ef database update -p ./HoneyDo.Infrastructure/ -s ./HoneyDo.Web/ -c HoneyDoContext`.
Also you need to add `JwtKey` as found in the `README.md`

File: README.md (hunk 1 above)
Comment (jantaylor) [reply README.md]: Those are the same for Linux.

**Pull request merged**